packages/app/src/SessionSubpanel.tsx

```diff
@@ -0,0 +1,637 @@
+      alias: 'id',
+    },
+    {
+      valueExpression: `${traceSource.eventAttributesExpression}['location.href']`,
+      alias: 'location.href',
+    },
+    {
+      valueExpression: 'ScopeName', // FIXME: add mapping
+      alias: 'otel.library.name',
+    },
+    {
+      valueExpression: `${traceSource.parentSpanIdExpression}`,
+      alias: 'parent_span_id',
+    },
+    {
+      valueExpression: `${traceSource.statusCodeExpression}`,
+      alias: 'severity_text',
+    },
+    {
+      valueExpression: `${traceSource.spanIdExpression}`,
+      alias: 'span_id',
+    },
+    {
+      valueExpression: `${traceSource.spanNameExpression}`,
+      alias: 'span_name',
+    },
+    {
+      valueExpression: `${traceSource.timestampValueExpression}`,
+      alias: 'timestamp',
+    },
+    {
+      valueExpression: `${traceSource.traceIdExpression}`,
+      alias: 'trace_id',
+    },
+    {
+      valueExpression: `CAST('span', 'String')`,
+      alias: 'type',
+    },
+  ];
+
+  const filteredEventsWhere = `
+    ${traceSource.resourceAttributesExpression}.rum.sessionId:"${rumSessionId}"
+    AND (
+      ${traceSource.eventAttributesExpression}.http.status_code:>299 
+      OR ${traceSource.eventAttributesExpression}.component:"error" 
+      OR ${traceSource.spanNameExpression}:"routeChange" 
+      OR ${traceSource.spanNameExpression}:"documentLoad" 
+      OR ${traceSource.spanNameExpression}:"intercom.onShow" 
+      OR ScopeName:"custom-action" 
+    ) ${searchedQuery}`;
+
+  const allEventsWhere = `
+    ${traceSource.resourceAttributesExpression}.rum.sessionId:"${rumSessionId}"
+    AND (
+      ${traceSource.eventAttributesExpression}.http.status_code:* 
+      OR ${traceSource.eventAttributesExpression}.component:"console" 
+      OR ${traceSource.eventAttributesExpression}.component:"error" 
+      OR ${traceSource.spanNameExpression}:"routeChange" 
+      OR ${traceSource.spanNameExpression}:"documentLoad" 
+      OR ${traceSource.spanNameExpression}:"intercom.onShow" 
+      OR ScopeName:"custom-action" 
+    ) ${searchedQuery}`;
+
+  const playBarEventsConfig = useMemo<ChartConfigWithOptDateRange>(
+    () => ({
+      select: commonSelect,
+      from: traceSource.from,
+      dateRange: [start, end],
+      // TODO: support SQL ? (should be consistent with searchQuery)
+      whereLanguage: 'lucene',
+      where: filteredEventsWhere,
+      timestampValueExpression: traceSource.timestampValueExpression,
+      implicitColumnExpression: traceSource.implicitColumnExpression,
+      connection: traceSource.connection,
+      orderBy: `${traceSource.timestampValueExpression} ASC`,
+      limit: {
+        limit: 4000,
+        offset: 0,
+      },
+    }),
+    [rumSessionId, start, end, searchedQuery],
+  );
+  const [playerFullWidth, setPlayerFullWidth] = useState(false);
+
+  const sessionEventListConfig = useMemo<ChartConfigWithOptDateRange>(
```

Comment:
converted v1 to v2 configs here `playBarEventsConfig` and `sessionEventListConfig`